### PR TITLE
test: Fix Windows\ImagesComparisonTests

### PR DESCRIPTION
**Pull request by Dor-bl**
## Change list

- Rename the class, since we have more than one test.
- Add AutomationName capability to support appium 2.x server
- Add LaunchApp() to improve stability on splash screen cases.
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [ ] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [x] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)
image

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: test/integration/Windows/ImagesComparisonTest.cs [hunk 1]
```diff
@@ -19,11 +19,13 @@ public void BeforeAll()
             appCapabilities.App = "Microsoft.WindowsCalculator_8wekyb3d8bbwe!App";
             appCapabilities.DeviceName = "WindowsPC";
             appCapabilities.PlatformName ="Windows";
+            appCapabilities.AutomationName= "Windows";
```
Comment (mykola-mokhnach) [inline test/integration/Windows/ImagesComparisonTest.cs:22]: missing space

File: test/integration/Windows/ImagesComparisonTest.cs [hunk 2]
```diff
@@ -19,11 +19,13 @@ public void BeforeAll()
...
 
             var serverUri = Env.ServerIsRemote() ? AppiumServers.RemoteServerUri : AppiumServers.LocalServiceUri;
             _calculatorSession = new WindowsDriver(serverUri, appCapabilities,
                 Env.InitTimeoutSec);
             _calculatorSession.Manage().Timeouts().ImplicitWait = Env.ImplicitTimeoutSec;
+            _calculatorSession.LaunchApp();
```
Comment (mykola-mokhnach) [inline test/integration/Windows/ImagesComparisonTest.cs:28]: shoudln't LaunchApp be marked as deprecated

Comment (Dor-bl) [reply test/integration/Windows/ImagesComparisonTest.cs:28]: I probably missed the memo, can you please refer me to the topic?

File: test/integration/Windows/ImagesComparisonTest.cs (hunk 1 above)
Comment (Dor-bl) [reply test/integration/Windows/ImagesComparisonTest.cs:22]: fixed the one above as well :)

File: test/integration/Windows/ImagesComparisonTest.cs (hunk 2 above)
Comment (Dor-bl) [reply test/integration/Windows/ImagesComparisonTest.cs:28]: @mykola-mokhnach  this one? https://github.com/appium/java-client/pull/1571

Comment (Dor-bl) [reply test/integration/Windows/ImagesComparisonTest.cs]: Removed it anyway

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**